### PR TITLE
Add compact GoalCard component --references #27

### PR DESCRIPTION
**Pull request by karolsojko**
Add compact GoalCard component --references #27

Only issues I'm having is to display the svg items - apparently after build the svg content is loaded properly in the css but it doesn't display - maybe @szymonmichalak or @jacekelgda could help with that

### CODE REVIEW

**Comment by jacekelgda** [comment]
I was not able to create proper svg either, we need a solution for this.

**Comment by jacekelgda** [comment]
This is only compact view ? Not extended ? 

Seems the progress bar is failing to show on firefox. Checked on the styleguides and it's very similar issue.

Do you know why the width of the card is different then in styleguide ?

**Comment by karolsojko** [comment]
@jacekelgda 

Yes, only compact - extended will be in another PR

It's different since the width is extended from the timeline div. I guess we will add timeline styles on the Unleash App itself - just like it was with the case of profile info

**Comment by jacekelgda** [comment]
@karolsojko I would actually be in favor of creating set of components in our living styleguide ( just as an example of usage ) where you collect all components that are in Profile section for example - just so we don't miss anything. You will not be creating one component that is `<ProfileSection />` but you can still arrange it in our doc.

I am doing to same approach for each component ( and components sets ) - create header, dropdown menus, tabs, unleash tempo etc.

Planning to do the same for timeline as well eventually.

File: src/components/GoalCard/GoalCard.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react'
```
Comment (jacekelgda) [inline src/components/GoalCard/GoalCard.js]: Pleas add flow

File: src/components/GoalCard/GoalCardItem.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
```
Comment (jacekelgda) [inline src/components/GoalCard/GoalCardItem.js]: Please add flow

**Comment by karolsojko** [comment]
@jacekelgda ok I'll put the `Timeline` as a separate component in another PR

**Comment by jacekelgda** [comment]
@karolsojko I actually created a separate issue for this. So maybe stick to creating extended version and then we can do the timeline in separate one - does it make sense ? Or a timeline can't be separated ? 

I actually thought we could do a `<Timeline />` component and each `<GoalCard />` component would be a child of it. That way the `<Timeline />` could crate the line etc. Just an idea, I didn't look into code yet.

**Pull request merged**